sfge/src/main/java/com/salesforce/graph/build/AbstractApexVertexBuilder.java

```diff
@@ -72,23 +73,37 @@ private void buildVertices(JorjeNode node, Vertex vNodeParam, String fileName) {
         }
 
         Vertex vPreviousSibling = null;
-        for (JorjeNode child : node.getChildren()) {
-            Vertex vChild = g.addV(child.getLabel()).next();
+        final List<JorjeNode> children = node.getChildren();
+        final Set<Vertex> verticesAddressed = new HashSet<>();
+        verticesAddressed.add(vNode);
+
+        for (int i = 0; i < children.size(); i++) {
+            final JorjeNode child = children.get(i);
+            final Vertex vChild = g.addV(child.getLabel()).next();
             addProperties(g, child, vChild);
-            // We are currently adding PARENT and CHILD, in theory the same edge could be navigated
-            // both ways, however
-            // the code looks messy when that is done.
-            // TODO: Determine if this causes performance or resource issues. Consider a single edge
-            g.addE(Schema.PARENT).from(vChild).to(vNode).iterate();
-            g.addE(Schema.CHILD).from(vNode).to(vChild).iterate();
+
+            // Handle static block if needed
+            if (StaticBlockUtil.isStaticBlockStatement(node, child)) {
+                final Vertex parentVertexForChild =
```

Comment:
It looks like what's happening is that we detect that the child is a static block, and if so we create our synthetic method block to hold it, make that a child of the current parent, and make the block statement a child of that. Am I reading that right?
I'd love some commenting about what is being done here, and why it was necessary.

Reply:
Added more comments. Let me know if it explains sufficiently or I'll try expanding it some more.